test/mini-oidc/go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/canonical/lxd/test/mini-oidc
 
-go 1.20
+go 1.21
+
+toolchain go1.22.0
```

Comment:
It seems strange that the go version is 1.21 but the toolchain version is 1.22. But as this is only for testing I don't think it warrants preventing the merge.